spec/jobs/perform_transfer_job_spec.rb

```diff
@@ -6,10 +6,16 @@
   let(:perform_transfer_job) { described_class.new }
   let(:aws_storage_provider) { FactoryBot.create(:storage_provider, :aws) }
   let(:pending_transfer) { FactoryBot.create(:pending_transfer, storage_provider: aws_storage_provider) }
+  # this is the hexed checksum of the pending transfer fixture
+  let(:well_known_checksum) { '31a961575a28515eb6645610a736b0465ef24f9105892e18808294afe70c00f6' }
   let(:object_key) { 'safe/object/key.jpg' }
   let(:object_key_as_base64) { Base64.strict_encode64(object_key) }
+  let(:metadata_key) { PerformTransferJob::ORIGINAL_PATH_METADATA_KEY }
```

Comment:
```suggestion
  let(:original_path_metadata_key) { PerformTransferJob::ORIGINAL_PATH_METADATA_KEY }
```
Suggesting this, plus associated updates later in the file.

Reply:
Just pushed a commit with this change.